test-project/Assets/Generated/Source/improbable/testschema/ExhaustiveSingularComponentReaderWriter.cs

```diff
@@ -1,6 +1,6 @@
-// ===========
+// =====================================================
```

Comment:
Why are all of these changed in the build system module update..?

Reply:
forgot to generate test-project before the final codewriter pr (where the header changed)

this PR is also not based off the test-project port PR  - hence the ===== in this and #1241